### ISSUE TITLE
Question - Is it possible to assign colours from global variable in grouping

### ISSUE THREAD

**Issue by stephen147** [reupen/columns_ui#336]
I have a global variable:
`$set_global(txtlightclr,$rgb(128,128,128))`

Is it possible to then assign it in the grouping but it says unknown function.

![foobar2000_OML0ysbw33](https://user-images.githubusercontent.com/14054664/91907845-7b8e4580-eca2-11ea-9f02-f4c2cc9533ff.png)

**Comment by reupen** [comment]
No, unfortunately for performance reasons global variables weren't made available in groups.

This is because all group titles are evaluated up front (i.e. when foobar2000 starts or you switch playlist), while playlist items titles are evaluated lazily (when they come into view).

**Comment by stephen147** [comment]
No worries, I'm just in the process off changing the theme to use global colours.

Would it be a big hit if there was an option to use it with the default, it been turned off?

**Comment by reupen** [comment]
No, there wouldn't be a performance hit if it were an option and that option was turned off.

The hit when the option is enabled would depend on things like the complexity of the global variables script and the number of tracks in the playlist.

**Comment by stephen147** [comment]
I meant that of you could add this functionality but have the default to it been off.

I wouldn't mind the performance hit. So it would be nice to have the option to enable this.
